moon/ros_workspace/src/aliencontrol/aliencontrol

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""The aliencontrol ROS node
```

Comment:
I like the node name :)

Reply:
'aliencontrol' is an external project - https://github.com/acschaefer/aliencontrol